tests/Unit/MartinGeorgiev/Doctrine/DBAL/Types/LtreeTest.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit\MartinGeorgiev\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\PostgreSQLPlatform;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidLtreeForDatabaseException;
+use MartinGeorgiev\Doctrine\DBAL\Types\Exceptions\InvalidLtreeForPHPException;
+use MartinGeorgiev\Doctrine\DBAL\Types\Ltree;
+use MartinGeorgiev\Doctrine\DBAL\Types\ValueObject\Ltree as LtreeValueObject;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+final class LtreeTest extends TestCase
+{
+    private MockObject&PostgreSQLPlatform $platform;
+
+    private Ltree $fixture;
+
+    protected function setUp(): void
+    {
+        $this->platform = $this->createMock(PostgreSQLPlatform::class);
+
+        $this->fixture = new Ltree();
+    }
+
+    #[Test]
+    public function has_name(): void
+    {
+        self::assertSame('ltree', $this->fixture->getName());
+    }
+
+    public function test_get_sql_declaration(): void
```

Comment:
The repository uses PHP attributes instead of the prefix convention for the tests. Can you please import the relevant PHPUnit `Test` class for it? Please take a look at the other test files around this one for examples.